src/app/recruit/[id]/edit/panel/BasicSelect.tsx

```diff
@@ -80,7 +80,7 @@ export default function BasicSelect({
     <Box>
       <FormControl fullWidth>
         <Select
-          sx={{ width: '416px', height: '32px' }}
+          sx={{ width: '26rem', height: '32px' }}
```

Comment:
px에서 rem으로 바꾸시다가 깜빡하신 것 같아 코멘트 남겨둡니다. 
```suggestion
          sx={{ width: '26rem', height: '2rem' }}
```